src/Microsoft.Diagnostics.Runtime.Tests/src/TestTargets.cs

```diff
@@ -90,12 +90,18 @@ public TestTarget(string name)
 
         private static DataTarget LoadDump(string path)
         {
-            DataTarget dt = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? DataTarget.LoadCrashDump(path) : DataTarget.LoadCoreDump(path);
-            dt.BinaryLocator = new SymbolServerLocator("");
-            return dt;
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
```

Comment:
Maybe I should remove "LoadCoreDump/LoadCrashDump" and just publish "LoadDump" and have the right reader selected based on either the operating system or the header of the file itself.  (The latter gets complicated because DbgEngDataReader allows for .zip and .cab files too, which is useful for internal Microsoft developers.)

Reply:
Any thoughts?

Reply:
In fact, some functionalities are still available for `LoadCoreDump` on Windows, which don't request information from DAC.

Reply:
Good point.  I'll leave this as-is for now.

Reply:
`LoadCoreDump` behaved well when I was working on #349 by experimenting with a coredump on Windows, but it won't work now because it throws after #393. Maybe we can throw that exception later when DAC is required.